man/systemd-sysext.xml

```diff
@@ -205,6 +212,51 @@
     to tie the most frequently configured options to runtime updateable flags that can be changed without a
     system reboot. This will help reduce servicing times when there is a need for changing the OS configuration.</para></refsect1>
 
+  <refsect1>
+    <title>Mutability</title>
+    <para>By default, merging system extensions on mutable host file systems will render <filename>/usr/</filename>
+    and <filename>/opt/</filename> hierarchies read-only. Merging configuration extensions will have the same
+    effect on <filename>/etc/</filename>. Mutable mode allows writes to these locations when extensions are
+    merged.</para>
+
+    <para>The following modes are supported:
+    <orderedlist>
+      <listitem><para><option>auto</option>: Automatic mode; the default. Mutability is disabled by default
```

Comment:
this bit is out of date

Reply:
See https://github.com/systemd/systemd/pull/31000#issuecomment-1953845283, will update today.